src/_pyinstaller_hooks_contrib/tests/test_libraries.py

```diff
@@ -1559,3 +1559,20 @@ def test_customtkinter(pyi_builder):
     pyi_builder.test_source("""
         import customtkinter
     """)
+
+
+@importorskip('pylibmagic')
+def test_pylibmagic(pyi_builder):
+    pyi_builder.test_source("""
+        import pylibmagic
+        import os
+
+        bundle_dir = os.path.normpath(sys._MEIPASS)
+        pylibmagic_data_path = f"{bundle_dir}/pylibmagic"
+
+        files_to_assert = ["libmagic.so.1", "magic.mgc"]
```

Comment:
Checking for `libmagic.so.1` will fail on macOS, because the corresponding shared lib is named `libmagic.1.dylib`.

Reply:
Thanks, @rokm. I was trying to figure out which lib the OS X used.

I will update the PR to include the respective compiled libs based on the OS.